includes/framework/QDbBackedSessionHandler.class.php

```diff
@@ -123,6 +123,15 @@ public static function SessionRead($id) {
 			if (!$result_row) // either the data was empty or the row was not found
 				return '';
 			$strData = $result_row['data'];
+
+			if(strstr($objDatabase->Adapter, 'PostgreSql')) {
```

Comment:
@spekary It would be wrong to say 
> it does not have good support for BLOB or binary columns

It does. It just does not handle them the way MySQL does. 

This is where we need to start working on per-database Data Types - except the basics (INT, FLOAT/REAL/DOUBLE, DATE, VARCHAR), most databases handle the other types differently. Yes, it can't be done *right now*. But we have to move in that direction with time.